### PR TITLE
move SimpleExpansionPanel to common utilities

### PR DESCRIPTION
**Pull request by sboleyn**
Lots of components that are not under components/tools/ are using that SimpleExpansionPanel. It should probably be moved into somewhere more common like components/utils/.

### CODE REVIEW

**Review by slr71 — APPROVED** [review]
I didn't notice any problems. 👍

File: src/components/utils/SimpleExpansionPanel.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
+/**
+ *
+ * A MUI accordinon that allows for displaying and
```
Comment (psarando) [inline src/components/utils/SimpleExpansionPanel.js:3]:
```suggestion
 * A MUI accordion that allows for displaying and
```

**Review by psarando — APPROVED** [review]
LGTM 👍 ✨

File: src/components/utils/SimpleExpansionPanel.js (hunk 1 above)
Comment (sboleyn) [reply src/components/utils/SimpleExpansionPanel.js:3]: Good catch!

**Pull request merged**